code/modules/custom_ka/upgrade_chips.dm

```diff
@@ -2,8 +2,10 @@
 	name = "upgrade chip - damage increase"
 	desc = "Increases damage and recoil, reduces accuracy."
 	icon_state = "upgrade_chip"
-	damage_increase = 10
+	damage_increase = 15
```

Comment:
Be careful when making jumps in KA chips since every component adds to the stats (barrels add to damage, too) Especially since for now, damage also affects AoE radius.

Reply:
Yes. That is why I coupled it with the firedelay_increase. The larger cores will make a good boom and decent damage, which is hard capped at 50 right now. You  could easily go past that by spaming the shots, but the delay should work around that.